Source/Shaders/GlobeFS.glsl

```diff
@@ -38,6 +38,10 @@ uniform float u_dayTextureOneOverGamma[TEXTURE_UNITS];
 uniform vec4 u_dayTextureTexCoordsRectangle[TEXTURE_UNITS];
 #endif
 
+#ifdef APPLY_COLOR_PALETTE
+uniform sampler2D u_dayTextureColorPalette[TEXTURE_UNITS];
```

Comment:
This is causing the WebGL warnings. There aren't as many color palettes as there are `TEXTURE_UNITS`. Define another constant for the number of color palettes and use that as the size of the array.